content/en/contribute/_index.md

```diff
@@ -7,111 +7,127 @@ menu:
     weight: 10
 ---
 
-The Cloud Native Glossary content is stored in [this GitHub repo](https://github.com/cncf/glossary) 
-where you'll find a list of [issues](https://github.com/cncf/glossary/issues), [PRs](https://github.com/cncf/glossary/pulls), and 
-[discussions](https://github.com/cncf/glossary/discussions) about the glossary. 
+## Welcome 
 
-There are four ways you can contribute:
+Welcome to the Cloud Native Glossary contributing guide and thank you for your interest. If you would like to contribute to our project, checkout the following list of contribution types and their corresponding sections in this guide: 
 
 1) [Work on an existing issue](#work-on-an-existing-issue)
 2) [Propose new terms](#propose-new-terms)
 3) [Update existing ones](#update-an-existing-term)
-4) [Help translate the glossary](#help-translate-the-glossary)
+4) [Localize the glossary](#help-translate-the-glossary)
+
+## CNCF glossary overview 
+
+The goal of this glossary is to simplify the cloud native space — which is notorious for its complexity — and thus make it more accessible to people. 
+
+The Cloud Native Glossary content is stored in [this GitHub repo](https://github.com/cncf/glossary) 
+where you'll find a list of [issues](https://github.com/cncf/glossary/issues), pull requests ([PRs](https://github.com/cncf/glossary/pulls)), and 
+[discussions](https://github.com/cncf/glossary/discussions) about the glossary. 
 
 ## Who can contribute?
 
-The goal of this Glossary is to make the cloud native space — which is notorious for its complexity — simpler for people by making it easier to understand. 
-To do that, we must focus on simplicity. 
+Basically, anyone can contribute, however, how you participate in the project depends on your level of expertise of the cloud native space. 
+Simplifying complex concepts requires a deep knowledge of the topic. Therefore, to contribute with a new term you must be proficient in the cloud native space. 
```

Comment:
```suggestion
Simplifying complex concepts requires a deep knowledge of the topic. 
Therefore, to contribute new terms, you must be proficient in them. 
Contributors are typically engineers who have worked with these technologies for some time or academics focused on cloud native. 

That know-how is required because explaining complex concepts in simple words is _really_ hard. And while the digestible, user-friendly outcome may seem easy, achieving the desired simplicity results from hard work and collaboration between cloud native experts. 
```

Moving that part up again to further explain that, just because the result looks easy, it meant it is easy.